lib/services/project-changes-service.ts

```diff
@@ -160,6 +161,7 @@ export class ProjectChangesService implements IProjectChangesService {
 			let prepareInfoFile = path.join(platformData.projectRoot, prepareInfoFileName);
 			this._outputProjectMtime = this.$fs.getFsStats(prepareInfoFile).mtime.getTime();
 			this._outputProjectCTime = this.$fs.getFsStats(prepareInfoFile).ctime.getTime();
+			this._outputProjectATime = this.$fs.getFsStats(prepareInfoFile).atime.getTime();
 			return false;
```

Comment:
so we start checking for changes: https://github.com/NativeScript/nativescript-cli/blob/fe95da72b8f3c93e9a124aa673fb25a7d8c10995/lib/services/project-changes-service.ts#L61